_data/simple-icons.json

```diff
@@ -10413,6 +10413,12 @@
             "hex": "3EA8FF",
             "source": "https://zenn.dev/mediakit"
         },
+        {
+            "title": "Zenodo",
+            "hex": "000000",
```

Comment:
> I set the hex value to `000000` because the primary colour of the logo is white on a linear gradient of `0047A8` to `2BBCFF`. Should I put either of these colours, or the "average", or something else?

We usually go with a calculated midpoint in case of gradients, in this case [that would be `#1682D4`](https://meyerweb.com/eric/tools/color-blend/#0047A8:2BBCFF:1:hex). As I'm not seeing a particularly prominent blue on the website, nor is any specific hex mentioned, I think we should go with that for now.